.github/workflows/internal-archive-release.yml

```diff
@@ -299,41 +290,48 @@ jobs:
+          aws codeartifact login --tool twine --repository ${{ secrets.AWS_REPOSITORY }} \
+          --domain ${{ secrets.AWS_DOMAIN }} --domain-owner ${{ secrets.AWS_DOMAIN_OWNER }} \
+          --region ${{ secrets.AWS_REGION }} --profile ${{ env.TEMP_PROFILE_NAME }}
 
-      - name: "Alter version in metadata of python package"
+      - name: "Get version in package and versions published to internal pypi"
         run: |
+          versions_published="$(aws codeartifact list-package-versions --repository ${{ secrets.AWS_REPOSITORY }} \
+          --domain ${{ secrets.AWS_DOMAIN }} --domain-owner ${{ secrets.AWS_DOMAIN_OWNER }} \
+          --region ${{ secrets.AWS_REGION }} --profile ${{ env.TEMP_PROFILE_NAME }} --format pypi \
+          --package dbt-${{ inputs.dbms_name }} --output json  \
+          --query 'versions[*].version' | jq -r '.[]' | grep "^${{ inputs.version_number }}" || true )"  # suppress pipefail only here
+
           version_file="dbt/adapters/${{ inputs.dbms_name }}/__version__.py"
-          setup_file="./setup.py"
+          
           version_in_file=$(grep -E 'version(: str)? =' "${version_file}" | cut -d '"' -f2)
-
-          # check the latest build of adapter code in our archive
-          versions_on_aws="$(aws codeartifact list-package-versions --repository ${{ secrets.AWS_REPOSITORY }} --domain ${{ secrets.AWS_DOMAIN }} --domain-owner ${{ secrets.AWS_DOMAIN_OWNER }} --region ${{ secrets.AWS_REGION }} --profile ${{ env.TEMP_PROFILE_NAME }} --format pypi --package dbt-${{ inputs.dbms_name }} --output json  --query 'versions[*].version' | jq -r '.[]' | grep "^${{ inputs.version_number }}" || true )"  # suppress pipefail only here
-
-          current_latest_version="$(echo "${versions_on_aws}" | sort -V | tail -n 1 )"
-
           echo "[Debug] version_in_file: ${version_in_file}"
-          echo "[Debug] current_latest_version: ${current_latest_version}"
+          
+          echo "CURRENT_PKG_VERSION=${version_in_file}" >> "$GITHUB_ENV"
+          echo "VERSIONS_PUBLISHED=$(echo "${versions[*]}"| tr '\n' ',')" >> "$GITHUB_ENV"
+          echo "VERSION_FILE=${version_file}" >> "$GITHUB_ENV"
+
+      - name: "Determine next internal release version"
+        id: get_next_internal_release_version
+        uses: "dbt-labs/dbt-release/.github/actions/next-internal-version@addPythonScriptComparison"
+        with:
+          version_number: $CURRENT_PKG_VERSION
+          versions_published: $VERSIONS_PUBLISHED
 
+      - name: "Update version in package and setup.py"
+        run: |
+          internal_release_version="${{ steps.get_next_internal_release_version.outputs.internal_release_version }}"
+          commit_sha="$(git rev-parse HEAD)"
+          setup_file="./setup.py"
```

Comment:
This is not true for `dbt-postgres` and `dbt-adapters`.

Reply:
Good catch, I've added a link to this comment for our ticket to add dbt-adapters to the internal release pipeline: https://github.com/dbt-labs/adapters-team-private/issues/149